source/RevitLookup/Core/ComponentModel/Descriptors/DocumentDescriptor.cs

```diff
@@ -71,4 +71,14 @@ IVariants ResolveGetAllUnusedElements()
         }
 #endif
     }
+    
+    public void RegisterExtensions(IExtensionManager manager)
+    {
+        if (!_document.IsFamilyDocument) return;
+        manager.Register(nameof(FamilySizeTableManager.GetFamilySizeTableManager), context =>
+        {
+            var result = FamilySizeTableManager.GetFamilySizeTableManager(_document, new ElementId(BuiltInParameter.RBS_LOOKUP_TABLE_NAME));
```

Comment:
Use **context** arg instead of **_document** field to avoid closure

Reply:
Fixed